production.Dockerfile

```diff
@@ -3,6 +3,57 @@
 #
 # Note: for 'posthog/posthog-cloud' remember to update 'prod.web.Dockerfile' as appropriate
 #
+
+#
+# Build the frontend artifacts
+#
+FROM node:16.15-alpine3.14 AS frontend
+
+WORKDIR /code
+
+COPY package.json yarn.lock ./
+RUN yarn config set network-timeout 300000 && \
+    yarn install --frozen-lockfile
+
+COPY frontend/ frontend/
+COPY ./bin/ ./bin/
+COPY babel.config.js tsconfig.json webpack.config.js ./
+RUN yarn build
+
+#
+# Build the plugin-server artifacts. Note that we still need to install the
+# runtime deps in the main image
+#
+FROM node:16.15-alpine3.14 AS plugin-server
+
+WORKDIR /code/plugin-server
+
+# Install python, make and gcc as they are needed for the yarn install
```

Comment:
Nit: also Python?

Reply:
As far as I could tell yes python is also needed. I could be wrong there though.